src/Georadix.Core/MathHelper.cs

```diff
@@ -7,6 +7,28 @@
     /// </summary>
     public class MathHelper
     {
+        /// <summary>
+        /// Calculates the great circle distance, in meters, between two latitude/longitude points.
+        /// </summary>
+        /// <param name="lat1">The latitude of the first point.</param>
+        /// <param name="long1">The longitude of the first point.</param>
+        /// <param name="lat2">The latitude of the second point.</param>
+        /// <param name="long2">The longitude of the second point.</param>
+        /// <returns>The value in meters.</returns>
+        public static double CalculateGreatCircleDistance(double lat1, double long1, double lat2, double long2)
+        {
+            var earthRadius = 6371e3;
+            var deltaLat = DegreesToRadians(lat2 - lat1);
```

Comment:
Do the deltas need to be absolute or it doesn't matter?

Reply:
Needs to be this way, but will add test cases inverting points to prove accuracy.